nemo_curator/utils/fuzzy_dedup_utils/shuffle_utils.py

```diff
@@ -83,7 +83,7 @@ def get_shuffle_part_ids_df(
     num_workers=0,
 ):
     sizes = agg_df[size_col].values
-    max_text_bytes_per_part = int(np.iinfo(np.int32).max // 1.2)
+    max_text_bytes_per_part = int(np.iinfo(np.int32).max * 3)
```

Comment:
Same suggestion here.

Reply:
We can make this configurable by making it an argument to the function, and adding the option to set it when creating the `_Shuffle` class. I'm a bit hesitant to add it as a config option to `FuzzyDedupConfig` overall since there are a lot of knobs already and these are more advanced options that complicate the high level API. What do you think?

Reply:
Like from a user perspective, I would rather have them use the more advanced option than modify the code, like we do here to deal with OOMs. Which are more probable, now that  we have long strings. 

Of course, We can do that later on, this is not a blocking review for the PR and we should land this in the code freeze today. 

Can you file an issue around this please.

Reply:
Done: https://github.com/NVIDIA/NeMo-Curator/issues/233